app/components/task/holder.hbs

```diff
@@ -1,16 +1,42 @@
 <div class="task-card">
   <Task-Details @task={{@task}} />
-  <div class="progress-bar-container">
-    <div class="progress-bar">
-      <input
-        id="input"
-        type="range"
-        name="percentCompleted"
-        value="{{@task.percentCompleted}}"
-        min="0" max="100" step="25"
-        {{on "change" this.onPercentageChange}}
-        {{on "change" (fn @onTaskUpdate @task.id)}}
-      />
+  {{#if (eq @task.status 'IN_PROGRESS')}}
```

Comment:
Can we use a known enum here please?

Reply:
Didn't get this point, couldn't find anything useful on Googling for `enum in ember` or `enum in handlebars`
Can you please provide some resource for this one?

Reply:
```js
const STATUSES = {
  IN_PROGRESS: 'IN_PROGRESS',
}

```